Add flag to print unfolded projections as cases (#16994)
coq/coq: doc/changelog/08-vernac-commands-and-options/16994-pr-unfolded-proj.rst

- **Added:**

>   flag :flag:`Printing Unfolded Projection As Match` (off by default) to be able to distinguish unfolded from folded primitive projections

Reviewer: ```suggestion
  flag :flag:`Printing Unfolded Projection As Match` to distinguish unfolded and folded primitive projections
```

No need to mention the default setting in the changelog since that's covered elsewhere.

Author: Nothing wrong with mentioning it though
